src/stravalib/client.py

```diff
@@ -654,6 +654,13 @@ def get_activity(
         class: `model.DetailedActivity`
             An Activity object containing the requested activity data.
 
+        Examples
+        --------
+        # Assuming an authenticated client
```

Comment:
we could setup doctest for these but i don't yet know how we'd do that without needing a token / proper authentication.

Reply:
An examples section in each client method's docstring seems to be overkill for this API. We already provide a link to the Strava API docs (which provide example responses). If particular parameters have a not-too-obvious effect on the method's behavior, we can always add examples on a per-instance basis.

As you mention, doctest wouldn't work here unless we're willing to jump through burning hoops to integrate doctest with our Strava API mock.